pkg/api/v1/schema.go

```diff
@@ -697,6 +698,12 @@ type Interface struct {
 	Ports []Port `json:"ports,omitempty"`
 	// Interface MAC address. For example: de:ad:00:00:be:af or DE-AD-00-00-BE-AF.
 	MacAddress string `json:"macAddress,omitempty"`
+	// BootOrder is an integer value > 0, used to determine ordering of boot devices.
+	// Lower values take precedence.
+	// Each interface or disk that has a boot order must have a unique value.
+	// Interfaces without a boot order are not tried.
+	// +optional
+	BootOrder *uint `json:"bootOrder,omitempty"`
```

Comment:
Please add a validation check to the validation webhook to make sure that it is unset or greater than 0.

Reply:
what about the validation that all orders are unique among interfaces and disks?

Reply:
Is this a libvirt requirement?

Reply:
I think it's not, but it's also quite useless/unpredictable otherwise. The boot order functionality is wonky on its own, perhaps best to ensure some basic sanity

Reply:
libivirt would fail if we have duplicate values

Reply:
Then let's catch this early.